community/deb.sh

```diff
@@ -79,10 +79,19 @@ detect_os ()
   echo "Detected operating system as $os/$dist."
 }
 
+add_pgdg ()
+{
+  echo "Registering PostgreSQL PGDG dependencies..."
+  echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
```

Comment:
I feel a _little_ uneasy blasting over this file without checking first. I know this script also blasts over the packagecloud file, but we "own" that one. This one could already be there on a user's machine.